### PR TITLE
feat: update storefront ui

### PR DESCRIPTION
**Pull request by zoltantakacs-born**
Update the storefroint ui package to latest version.

## Related Issue
VBC-500

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by scernisevs — APPROVED** [review]
LGTM

File: packages/theme/pages/Category.vue [hunk 1]
```diff
@@ -291,6 +336,7 @@ export default defineComponent({
     'stale-when-revalidate': 5
   }),
   setup() {
+    const isMobile = ref(mapMobileObserver().isMobile.get());
```
Comment (WojtekTheWebDev) [inline packages/theme/pages/Category.vue:339]: imo it should be `computed`

**Review by WojtekTheWebDev — CHANGES_REQUESTED** [review]
@zoltantakacs-born 🙌  Only one change with isMobile

File: packages/theme/components/RelatedProducts.vue [hunk 2]
```diff
@@ -70,13 +77,19 @@ import {
...
+  PropType,
+  ref
+} from '@nuxtjs/composition-api';
 import { useProductData } from '../composables/useProductData';
 import { useWishlistData } from '../composables/useWishlistData';
 
 export default defineComponent({
   name: 'RelatedProducts',
   setup() {
+    const isMobile = ref(mapMobileObserver().isMobile.get());
```
Comment (WojtekTheWebDev) [inline packages/theme/components/RelatedProducts.vue:92]: And here also `computed`
```suggestion
    const isMobile = computed(() => mapMobileObserver().isMobile.get());
```

**Review by WojtekTheWebDev — CHANGES_REQUESTED** [review]
One more

File: packages/theme/pages/Category.vue (hunk 1 above)
Comment (zoltantakacs-born) [reply packages/theme/pages/Category.vue:339]: yeah, I thought the same. But I checked how it is used inside VueStoreFront and just copied that solution:
https://github.com/vuestorefront/vue-storefront/blob/cf78686ad9ca2634cf0e2f4f613992e099cca078/packages/core/nuxt-theme-module/theme/components/AppHeader.vue

**Review by WojtekTheWebDev — APPROVED** [review]
LGTM

**Pull request merged**